docker/consul-template.service

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+exec consul-template \
+     -consul=consul:8500 \
+     -template "/etc/consul-templates/nginx.conf:/etc/nginx/nginx.conf:sv hup nginx"
```

Comment:
[sv](http://smarden.org/runit/sv.8.html) "control and manage services monitored by [runsv](http://smarden.org/runit/runsv.8.html)"